src/text.c

```diff
@@ -161,6 +162,15 @@ main(int argc, char *argv[])
 				}
 			}
 			break;
+		case 'R':
```

Comment:
Let's define a negative version, it's necessary for if there's a default in the axel config file. I'm not sure a short version is needed (I've been thinking about removing support for systems not supporting getopt_long anyway).

Reply:
About negative version of the option, I don't know whether it is a good idea to make .netrc parsing default, as it may break existing users' assumptions/scripts/workflows, etc. What do you think about it?

Reply:
I don't mean to make it the default, but if you have the option on the configuration file, then you need a way to disable it from the command line.

Reply:
Ok, I'll add it.